### ISSUE TITLE
第一周 视频公开课【草稿】

### ISSUE THREAD

**Issue by idisblueflash** [DerekGrant/Union-A---XLab#23]
## 目的
- 排查有漏洞的学员
- 夯实学员基础

## 预备选项
- Backpropagation 讲解
- Gradient Descent 讲解
- 准备相关问答引发学生思考，同时检验学生了解程度
- 提供相关具体推导视频，不占用公开课时间
  - [Gradient DescentMIT公开课视频](http://open.163.com/movie/2010/8/C/O/M6TUC9K75_M6TUIFJCO.html)
   - [Coursera微积分基础](https://www.coursera.org/learn/calculus1)
   - [微积分入门什么书籍好？](https://www.zhihu.com/question/32729130) 
     - [Multivariable Calculus by James Stewart](https://www.amazon.cn/图书/dp/1305266641/ref=sr_1_6?s=books&ie=UTF8&qid=1502326921&sr=1-6)
## 背景
一对一聊过后，很多同学对上面内容掌握不牢，表现为：
1. 了解不透
1. 不懂推导

## 希望
- 各位有好的建议，资料，回复整理过来。
- 有不同意见，请尽量写出来，讨论
